### PR TITLE
Db 7 - Account

### PR DESCRIPTION
**Pull request by Giovanni-Schroevers**
<h1>DB-7 - Account</h1>

- Added Progress bar
- Added Icons and text to show objectives
- Added Rewardes box
- Positioned account box in grid

### CODE REVIEW

File: Default/public/stylesheets/sass/style.scss [hunk 1]
```diff
@@ -259,4 +260,129 @@ body {
...
+  padding-left: 0;
+}
+.account__item {
+  float: left;
+  padding-left: 0;
+  font-size: 14px;
+  text-align: center;
+  width: 16%;
+}
+.account__item--2 {
```
Comment (MatthijsHoek) [inline Default/public/stylesheets/sass/style.scss]: Geen BEM

File: Default/public/index.html [hunk 2]
```diff
@@ -74,6 +77,49 @@ <h3 class="notif__text--h3">Congratulation Jennie,</h3>
...
+                <div class="account__gradient"></div>
+            </div>
+
+            <div class="account__info">
+                <ul class="account__list">
+                    <li class="account__item account__item--3">
+                        <i class="far fa-user"></i>
+                        <p>User Details</p>
+                    </li>
+                    <li class="account__item account__item--2 account__item--3">
```
Comment (MiguelFieira) [inline Default/public/index.html]: Account__item i'sent BEM

File: Default/public/stylesheets/sass/style.scss [hunk 3]
```diff
@@ -259,4 +260,129 @@ body {
...
+  margin-left: 4%;
+}
+.account__item--3 {
+  color: $blue;
+}
+
+.account__item--4 {
+  color: #C4C4C4;
+}
+.account__points-box{
```
Comment (MatthijsHoek) [inline Default/public/stylesheets/sass/style.scss]: Add one more "-" to modifier

File: Default/public/stylesheets/sass/style.scss [hunk 4]
```diff
@@ -259,4 +260,129 @@ body {
...
+  width: 16%;
+}
+.account__item--2 {
+  margin-left: 4%;
+}
+.account__item--3 {
+  color: $blue;
+}
+
+.account__item--4 {
```
Comment (MatthijsHoek) [inline Default/public/stylesheets/sass/style.scss]: Geen BEM

File: Default/public/stylesheets/sass/style.scss [hunk 5]
```diff
@@ -259,4 +260,129 @@ body {
...
+  bottom: -10px;
+  left: 35%;
+}
+.account__points-text{
+  color: white;
+  font-size: 67%;
+  font-family: $font;
+  margin-left: 3%;
+}
+.account__points-box--small{
```
Comment (MatthijsHoek) [inline Default/public/stylesheets/sass/style.scss]: New selector name

File: Default/public/stylesheets/sass/style.scss [hunk 6]
```diff
@@ -259,4 +260,129 @@ body {
...
+  position: absolute;
+  border-radius: 2px;
+  background: #034193;
+  width: 42%;
+  height: 75%;
+  right: 2%;
+  top: 12.5%;
+}
+.account__points-text--small{
```
Comment (MatthijsHoek) [inline Default/public/stylesheets/sass/style.scss]: New selector name

File: Default/public/stylesheets/sass/style.scss [hunk 7]
```diff
@@ -259,4 +260,129 @@ body {
...
+  top: 13%;
+  left:15.5%;
+}
+.account__arrow {
+  width: 0;
+  height: 0;
+  border-left: 10px solid transparent;
+  border-right: 10px solid transparent;
+
+  border-top: 10px solid $blue;
```
Comment (MatthijsHoek) [inline Default/public/stylesheets/sass/style.scss]: Remove enter above and under

File: Default/public/stylesheets/style.css [hunk 8]
```diff
@@ -172,13 +207,156 @@ body {
...
+  position: absolute;
+  border-radius: 2px;
+  background: #034193;
+  width: 42%;
+  height: 75%;
+  right: 2%;
+  top: 12.5%;
+}
+
+.account__points-text--small {
```
Comment (MatthijsHoek) [inline Default/public/stylesheets/style.css]: New selector name

**Pull request closed without merging**